js/repl/compile.js

```diff
@@ -137,6 +138,8 @@ export default function compile(code: string, config: CompileConfig): Return {
     };
 
     const transformed = Babel.transform(code, babelConfig);
+
+    astContext = JSON.stringify(transformed.ast);
```

Comment:
The problem with this is the same as also with #1421. The ast is for the transformed output which is confusing imho. So for example if the input is `() => {}` and preset-env is enabled there is no ArrowFunction in the ast instead there is a FunctionExpression.

I wonder if there is a usecase were people want to see the transformed output, but atm I think it is more confusing. So I would prefer doing `Babel.parse` here when the ast panel is active (and also disabling all transform options in the interface maybe?)

But not sure what other people think?

Reply:
I agree :+1:
Maybe we could move the "AST" option to the top and make it a dropdown?

![schermata da 2019-02-10 00-05-33](https://user-images.githubusercontent.com/7000710/52527354-b9460a80-2cc7-11e9-9ff6-e78f8e30a9df.jpg)